test/command/suite/sharding/logical_range_filter/columns/stage/filtered/window_function/shards.expected

```diff
@@ -109,6 +111,35 @@ logical_range_filter Logs   --shard_key timestamp   --max '2017/03/17 10:00:00.0
     ]
   ]
 ]
+#|-| [obj][open] <273>(<Logs_20170315.timestamp>):<64>(<column:fix_size>)
+#|-| [obj][open] <272>(<Logs_20170315>):<51>(<table:no_key>)
+#|-| [obj][open] <276>(<Logs_20170316.timestamp>):<64>(<column:fix_size>)
+#|-| [obj][open] <275>(<Logs_20170316>):<51>(<table:no_key>)
```

Comment:
This isn't closed.
Is this closed by #1333 and #1334, right?

Reply:
I have rebased onto `master` and updated tests and confirmed it is closed.